kernel/syscall-defs.h

```diff
@@ -4,20 +4,14 @@
 typedef unsigned long long syscall_arg;
 
 typedef struct {
```

Comment:
This is no longer part of the ipc ABI, so I don't think it belongs here.  This probably means you have to define it twice, which is kind of lame.

Reply:
Done. I did this in the 'stop using memory in the ABI' commit, so this comment might get clobbered.